src/test/resources/s3-valid-inbound-cases.xml

```diff
@@ -66,6 +66,22 @@
 		<integration:poller fixed-rate="1000"/>
 	</int-aws:s3-inbound-channel-adapter>
 
+	<int-aws:s3-inbound-channel-adapter
+			id="validInboundWithProxy"
+			bucket="TestBucket"
+			propertiesFile="testawscredentials.properties"
+			temporary-suffix=".temp"
+			local-directory="${foo.dir:/}"
+			remote-directory="remote"
+			accept-sub-folders="true"
+			max-objects-per-batch="100"
+			file-name-regex="[A-Za-z0-9]+\\.txt"
+			channel="inbound"
+			auto-startup="false"
+            proxyHost="localhost"
+            proxyPort="80">
```

Comment:
Maybe. I haven't dug so deep. And just because I'd like to *deprecate* that `AmazonS3Operations` abstraction in favor of direct `AmazonS3Client` usage. 
That's why I say jsut make all those required changes (a new `AmazonS3Operations` implementation with `ClientConfiguration`)  locally and don't bring them to the Framework.

Looks like you should implement your own ctor there:
````java
s3Factory = new AbstractAWSClientFactory<AmazonS3Client>() {
			@Override
			protected AmazonS3Client getClientImplementation() {
				String accessKey = credentials.getAccessKey();
				String secretKey = credentials.getSecretKey();
				BasicAWSCredentials credentials = new BasicAWSCredentials(accessKey, secretKey);
				return new AmazonS3Client(credentials);
			}

		};
````
to inject supplied `ClientConfiguration` to that `AmazonS3Client` alongside with the `credentials`. I mean use this ctor:
````java
public AmazonS3Client(AWSCredentials awsCredentials, ClientConfiguration clientConfiguration) {
````

That's why the reusing the AWS API in our code (Spring Integration) would be better. You always can modify the `AmazonS3Client` as you want and need and just inject the final bean to the `<int-aws:s3-inbound-channel-adapter>` without any request, changes and waiting for release around Framework!